src/sqlfluff/dialects/dialect_postgres.py

```diff
@@ -1335,6 +1335,232 @@ class AlterTableActionSegment(BaseSegment):
+    """A `ALTER MATERIALIZED VIEW` statement.
+
+    As specified in https://www.postgresql.org/docs/14/sql-altermaterializedview.html
+    """
+
+    type = "alter_materialized_view_statement"
+
+    match_grammar = StartsWith(Sequence("ALTER", "MATERIALIZED", "VIEW"))
+
+    parse_grammar = Sequence(
+        "ALTER",
+        "MATERIALIZED",
+        "VIEW",
+        OneOf(
+            Sequence(
+                Sequence("IF", "EXISTS", optional=True),
+                Ref("TableReferenceSegment"),
+                OneOf(
+                    Delimited(Ref("AlterMaterializedViewActionSegment")),
+                    Sequence(
+                        "RENAME",
+                        Sequence("COLUMN", optional=True),
+                        Ref("ColumnReferenceSegment"),
+                        "TO",
+                        Ref("ColumnReferenceSegment"),
+                    ),
+                    Sequence("RENAME", "TO", Ref("TableReferenceSegment")),
+                    Sequence("SET", "SCHEMA", Ref("ParameterNameSegment")),
+                ),
+            ),
+            Sequence(
+                Ref("TableReferenceSegment"),
+                Ref.keyword("NO", optional=True),
+                "DEPENDS",
+                "ON",
+                "EXTENSION",
+                Ref("ParameterNameSegment"),
+            ),
+            Sequence(
+                "ALL",
+                "IN",
+                "TABLESPACE",
+                Ref("TableReferenceSegment"),
+                Sequence(
+                    "OWNED",
+                    "BY",
+                    Delimited(Ref("ObjectReferenceSegment")),
+                    optional=True,
+                ),
+                "SET",
+                "TABLESPACE",
+                Ref("ParameterNameSegment"),
+                Sequence("NOWAIT", optional=True),
+            ),
+        ),
+    )
+
+
+@postgres_dialect.segment()
+class AlterMaterializedViewActionSegment(BaseSegment):
+    """Alter Materialized View Action Segment.
+
+    Matches the definition of action in https://www.postgresql.org/docs/14/sql-altermaterializedview.html
+    """
+
+    type = "alter_materialized_view_action_segment"
+
+    match_grammar = OneOf(
+        Sequence(
+            "ALTER",
+            Ref.keyword("COLUMN", optional=True),
+            Ref("ColumnReferenceSegment"),
+            OneOf(
+                Sequence("SET", "STATISTICS", Ref("NumericLiteralSegment")),
+                Sequence(
+                    "SET",
+                    Bracketed(
+                        Delimited(
+                            Sequence(
+                                Ref("ParameterNameSegment"),
+                                Sequence(
+                                    Ref("EqualsSegment"),
+                                    Ref("LiteralGrammar"),
+                                    optional=True,
+                                ),
+                            ),
```

Comment:
```suggestion
                            Sequence(
                                Ref("ParameterNameSegment"),
                                Ref("EqualsSegment"),
                                Ref("LiteralGrammar"),
                            ),
```
value is not optional here
![image](https://user-images.githubusercontent.com/80432516/145086335-fdbce0bb-a35b-4d55-aeba-3352b466d445.png)